src/cc/arduino/plugins/wifi101/flashers/java/SSLCertDownloader.java

```diff
@@ -82,7 +82,7 @@ public void checkServerTrusted(X509Certificate[] xcs, String string) throws Cert
 		connection.setSSLSocketFactory(ssl.getSocketFactory());
 
 		Certificate[] certificates = null;
-		if (connection.getResponseCode() == 200) {
+		if (connection.getResponseCode() > 0) {
```

Comment:
I think so, we really just need a SSL connection. However, let's stick to `HttpsURLConnection` for now.